.travis.yml

```diff
@@ -0,0 +1,103 @@
+sudo: required
+dist: trusty
+
+os: linux
+
+language: c
+
+compiler:
+  - gcc
+
+env:
+  global:
+    - JOBS=3
+    - MBEDTLS_VERSION=2.2.1
+    - MBEDTLS_PREFIX=~/opt/mbed
+    - MBEDTLS_LIBS="-L$MBEDTLS_PREFIX/lib -lmbedtls -lmbedx509 -lmbedcrypto"
+    - MBEDTLS_CFLAGS="-I$MBEDTLS_PREFIX/include"
+    - OPENSSL_VERSION=1.0.1t
+    - OPENSSL_PREFIX=~/opt/ssl
```

Comment:
Hi,

On Tue, Jun 07, 2016 at 12:45:38AM -0700, Samuli Seppänen wrote:
> > +    - OPENSSL_VERSION=1.0.1t
> > +    - OPENSSL_PREFIX=~/opt/ssl
> 
> Yeah, /opt would do, because extracting the tarballs will create the necessary subdirectories anyways.
> 

~/opt is not /opt

gert
-- 
USENET is *not* the non-clickable part of WWW!
                                                           //www.muc.de/~gert/
Gert Doering - Munich, Germany                             gert@greenie.muc.de
fax: +49-89-35655025                        gert@net.informatik.tu-muenchen.de